exercises/reverse-string/canonical-schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "exercise": "reverse-string",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "detects reverse-string",
```

Comment:
I disagree with `detects`. I don't think the function under test is detecting anything. It is reversing a string. The description should reflect that.